specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -380,17 +380,14 @@
         }
       }
     },
-    "/{scope}/providers/Microsoft.ManagedServices/marketplaceRegistrationDefinitions": {
```

Comment:
suggest to have {scopeOptional} as this API supports tenant and subscription level as well.

Reply:
agree with removing; as I see for operations API also we have removed it.